### ISSUE TITLE
Backspace

### ISSUE THREAD

**Issue by nickgermaine** [liri-project/liri-browser#183]
Backspace key doesn't work in master at the moment.  When I'm in a text field, hitting backspace doesn't delete last character, it goes back a page.

**Comment by tim-sueberkrueb** [comment]
Hmm, this seems harder to fix than I thought. Looks like there's no simple way to get whether there's `activeFocus` on `WebEngineView`.

**Comment by Naizugai** [comment]
I have a problem where backspace does not go back a page.

**Comment by ViktorNova** [comment]
This behavior seems new, I haven't experienced this until today.. sadly it makes Liri unusable. An option do disable the backspace=back bahavior would be a great workaround for now!

**Comment by pierremtb** [comment]
@ViktorNova I have just removed it

**Comment by VirusKA** [comment]
yeah 0.5 issue

**Comment by tim-sueberkrueb** [comment]
Yeah, I'm sorry for this, this was my fault. See #171. This should be fixed in v0.4 (if possible) though. Will look into it. If it turns out this is an actual blocker we'll change the assignment to v0.5.

**Comment by pierremtb** [comment]
Maybe we could fix it for v0.4 using another shortcut, sth like CTRL+B(ack), if we can't find a quick way to solve it.
